Makefile

```diff
@@ -8,6 +8,8 @@ COMMON_DEPS=$(wildcard ./internal/**/*.go) go.sum go.mod
 
 APEX_DEPS=$(COMMON_DEPS) $(wildcard cmd/apex/*.go)
 
+APISERVER_DEPS=$(COMMON_DEPS) $(wildcard cmd/apiserver/*.go)
```

Comment:
I don't think this is required either now.

Reply:
It’s used as dependencies for the go-lint target, so that’s why I added it back. On second thought though I don’t think the dependencies are useful there since go-lint is a PHONY target.